### ISSUE TITLE
Issue installing phpMyAdmin

### ISSUE THREAD

**Issue by bsussman** [goodguyry/dreambox#38]
We use mysql successfully on our dreambox  VM.  I successfully  set  up  remote phpmyadmin to access  databases on this VM but cannot get phpmyadmin installed in the VM for folks unable to configure their phpmyadmin to talk to the VM.

**Steps taken:**

ssh to the running VM
```
sudo apt-get update
sudo apt-get dist-upgrade
```
apparently succeed
`sudo apt-get install phpmyadmin` fails in the canned install process, trying to connect to mysql.

Is there something special about installing phpmyadmin on dreambox?

**Comment by goodguyry** [comment]
Hey, so sorry for the delay! I'm just now seeing this...

The first thing that comes to mind is wondering if you've followed the steps in the Wiki to complete the [MySQL setup](../wiki#mysql-setup-optional)?

If so, I'll need to dig into this and see what else I can come up with. That is, unless you've figured it out in the time since you initially opened this 😖

**Comment by acodispo** [comment]
```
vagrant up
vagrant ssh
sudo apt-get update
sudo apt-get dist-upgrade
mysql_secure_installation
sudo apt-get install phpmyadmin
```

The following error occurs during phpMyAdmin configuration:

ERROR 2002 (HY000): Can't connect to local MySQL server through socket '/var/run/mysqld/mysqld.sock' (2) 

I can give more detail on the install process if you need.

**Comment by goodguyry** [comment]
This is fixed in v0.3.0
